workflows/workflows/bamtogff.wdl

```diff
@@ -6,115 +6,109 @@ workflow bamtogff {
         File bamindex
         File? control_bamfile
         File? control_bamindex
-
         File gtffile
         File chromsizes
-        Int distance = 2000 #distance from site
-
-        String samplename = basename(bamfile,'.bam')
+        Int distance = 2000  # distance from site
```

Comment:
@a-frantz - `format` added a preceding space, but missed the space after `#`.

Reply:
Yea, we don't add spaces after `#` signs, but I think we do have a lint rule that fires suggesting those get a space?

Reply:
Yeah, we have a lint rule for it, so I assumed that we intended `format` to insert those as well.

Reply:
I think there are enough edge cases that inserting spaces would be error-prone/annoying. It would be easy to detect our own lint directives (`#@`), but I think there are other cases where users _want_ no spaces